Framework/DataHandling/src/LoadILLTOF.cpp

```diff
@@ -549,27 +547,22 @@ void LoadILLTOF::loadDataIntoTheWorkSpace(
   std::vector<detid_t> monitorIDs = instrument->getMonitors();
 
   for (const auto &monitor : monitors) {
-    m_localWorkspace->dataX(spec)
-        .assign(detectorTofBins.begin(), detectorTofBins.end());
     // Assign Y
-    m_localWorkspace->dataY(spec).assign(monitor.begin(), monitor.end());
+    m_localWorkspace->mutableY(spec).assign(monitor.begin(), monitor.end());
     // Assign Error
-    MantidVec &E = m_localWorkspace->dataE(spec);
+    auto &E = m_localWorkspace->mutableE(spec);
     std::transform(monitor.begin(), monitor.end(), E.begin(),
                    LoadILLTOF::calculateError);
```

Comment:
I think this is just computing the `sqrt` of the `Counts`. In that case `Histogram` can deal with that automatically and you should be able to replace the whole block of code with something like this:

```cpp
m_localWorkspace->setHistogram(spec, BinEdges(m_localWorkspace->binEdges(0)), Counts(monitor.begin(), monitor.end()); // NOTE: No E specified so it defaults to sqrt(Counts)
```